File: 参考/comp9041-454/9041-labonserver/lab07/count_word.py
#!/usr/bin/python
import sys
import re
word=sys.argv[1]
word_len=0
for line in sys.stdin:
    list=re.findall(r'\b{}\b'.format(word),line,flags=re.IGNORECASE)
    word_len+=len(list)
print('{} occurred {} times'.format(word,word_len))

File: lab/lab07/count_word.pl
#!/usr/bin/perl -w
$target = lc $ARGV[0];
$count = 0;
while ($line = <STDIN>) {
  @arr = $line =~ /[a-zA-Z]+/ig;
  foreach $word (@arr) {
    $count++ if lc $word eq $target;
  }
}
print "$target occurred $count times\n";
